### PR TITLE
Add a template for reporting misalignment

### PR DESCRIPTION
**Pull request by matejak**
#### Description:

Provide a template with reasonable boilerplate description of a misalignment issue

#### Rationale:

Misalignment is a frequent problem, and making it easy to understand is beneficial.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/misalignment.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+name: Content Misalignment issue
+about: Report a state when this content is misaligned with a content produced by another party.
+---
+#### Description of problem:
+
+The content is misaligned with another content that targets the same policy - typically, this means that the system hardened by our content doesn't pass the scan by the other content.
```
Comment (yuumasato) [inline .github/ISSUE_TEMPLATE/misalignment.md:7]:
```suggestion
The content is misaligned with an external (third party) content that targets the same policy - typically, this means that a system hardened by our content doesn't pass the scan by the external content.
```

File: .github/ISSUE_TEMPLATE/misalignment.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+
+#### Details:
+
+This content is not aligned with content from _<insert the other party name here>_
+
+The misalignment affects this content's profiles
```
Comment (yuumasato) [inline .github/ISSUE_TEMPLATE/misalignment.md:14]:
```suggestion
The misalignment affects these profiles:
```

File: .github/ISSUE_TEMPLATE/misalignment.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+- ...
+
+The misalignment affects this content's rules
```
Comment (yuumasato) [inline .github/ISSUE_TEMPLATE/misalignment.md:18]:
```suggestion
The misalignment affects these rules:
```

File: .github/ISSUE_TEMPLATE/misalignment.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+- ...
+
+
+#### Outcome:
+
+- [ ] There is something to in this project to improve the content.
+  - [ ] Check needs to be improved.
+  - [ ] Remediation needs to be improved.
+- [ ] The other content's check is faulty - other party needs to be notified, they have work to do.
```
Comment (yuumasato) [inline .github/ISSUE_TEMPLATE/misalignment.md:28]:
```suggestion
- [ ] The external content's check is faulty - the other party needs to be notified, they have work to do.
```

File: .github/ISSUE_TEMPLATE/misalignment.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+
+#### SCAP Security Guide Version:
+
+#### The Other Policy Version:
```
Comment (yuumasato) [inline .github/ISSUE_TEMPLATE/misalignment.md:33]: Here you probably meant the version of the other party's content implementation. Not the Policy both contents are targeting, right?
```suggestion
####  External Content's Version:
```

File: .github/ISSUE_TEMPLATE/misalignment.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+- ...
+
+The misalignment affects this content's rules
+
+- ...
+
+
+#### Outcome:
+
+- [ ] There is something to in this project to improve the content.
```
Comment (yuumasato) [inline .github/ISSUE_TEMPLATE/misalignment.md:25]:
```suggestion
- [ ] This project's content can be  improved:
```

**Review by yuumasato — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/misalignment.md (hunk 5 above)
Comment (matejak) [reply .github/ISSUE_TEMPLATE/misalignment.md:33]: Correct.

**Review by yuumasato — APPROVED** [review]

**Pull request merged**